### PR TITLE
refactor(ast): remove `TSEnumBody`

### PR DESCRIPTION
**Pull request by Boshen**

### CODE REVIEW

**Comment by Boshen** [comment]
Current dependencies on/for this PR:
* **#2509** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/2509?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/2509?utm_source=stack-comment).

**Comment by Boshen** [comment]
### Merge activity

* **Feb 25, 11:44 PM EST**: @Boshen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/2509).

File: crates/oxc_ast/src/ast/ts.rs [hunk 1]
```diff
@@ -41,22 +41,10 @@ pub struct TSEnumDeclaration<'a> {
...
     pub id: BindingIdentifier,
-    #[cfg_attr(feature = "serde", serde(flatten))]
-    pub body: TSEnumBody<'a>,
+    pub members: Vec<'a, TSEnumMember<'a>>,
     /// Valid Modifiers: `const`, `export`, `declare`
     pub modifiers: Modifiers<'a>,
 }
-/// Enum Body
-///
-/// A scope must be created on the enum body so this abstraction exists
```
Comment (magic-akari) [inline crates/oxc_ast/src/ast/ts.rs:51]: Any updates?

Comment (Boshen) [reply crates/oxc_ast/src/ast/ts.rs:51]: I inlined enter / leave scope into the visitor function for now.

**Pull request merged**